corehq/apps/app_manager/static/app_manager/js/summary/form_summary.js

```diff
@@ -25,6 +25,11 @@ hqDefine('app_manager/js/summary/form_summary',[
                             questionIsVisible = questionIsVisible || _.find(question.options, function (option) {
                                 return match(query, option.value + self.translateQuestion(option));
                             });
+                            var casePropsVisible = _.find(question.load_properties.concat(question.save_properties), function (prop) {
+                                return match(query, prop[0] + prop[1]);
```

Comment:
Nitpick: `match(query, prop[0]) || match(query, prop[1])` would be a little more semantic.